containerized_lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
```

Comment:
I would place this script somewhere in the docs, so ppl will know that they can use it